python/ray/autoscaler/cli_logger.py

```diff
@@ -415,96 +452,27 @@ def doassert(self, val, msg, *args, **kwargs):
 
         For other arguments, see `_format_msg`.
         """
-        if self.old_style:
-            return
-
         if not val:
-            self.abort(msg, *args, **kwargs)
+            exc = None
+            if self.non_interactive:
+                exc = AssertionError()
 
-    def old_debug(self, logger, msg, *args, **kwargs):
-        """Old debug logging proxy.
-
-        Pass along an old debug log iff new logging is disabled.
-        Supports the new formatting features.
-
-        Args:
-            logger (logging.Logger):
-                Logger to use if old logging behavior is selected.
-
-        For other arguments, see `_format_msg`.
-        """
-        if self.old_style:
-            logger.debug(
-                _format_msg(msg, *args, **kwargs), extra=_parent_frame_info())
-            return
+            # TODO(maximsmol): rework asserts so that we get the expression
+            #                  that triggered the assert
+            #                  to do this, install a global try-catch
+            #                  for AssertionError and raise them normally
+            self.abort(msg, *args, exc=exc, **kwargs)
 
+    def old_debug(self, logger, msg, *args, **kwargs):
+        return
     def old_info(self, logger, msg, *args, **kwargs):
-        """Old info logging proxy.
-
-        Pass along an old info log iff new logging is disabled.
-        Supports the new formatting features.
-
-        Args:
-            logger (logging.Logger):
-                Logger to use if old logging behavior is selected.
-
-        For other arguments, see `_format_msg`.
-        """
-        if self.old_style:
-            logger.info(
-                _format_msg(msg, *args, **kwargs), extra=_parent_frame_info())
-            return
-
+        return
     def old_warning(self, logger, msg, *args, **kwargs):
-        """Old warning logging proxy.
-
-        Pass along an old warning log iff new logging is disabled.
-        Supports the new formatting features.
-
-        Args:
-            logger (logging.Logger):
-                Logger to use if old logging behavior is selected.
-
-        For other arguments, see `_format_msg`.
-        """
-        if self.old_style:
-            logger.warning(
-                _format_msg(msg, *args, **kwargs), extra=_parent_frame_info())
-            return
-
+        return
```

Comment:
should we also just remove the existing logger.old messages?

Reply:
yeah I'll add that to this PR too, this push is more of a checkpoint